derive/src/multihash.rs

```diff
@@ -1,12 +1,13 @@
 use std::collections::HashSet;
 
 use crate::utils;
-use proc_macro2::TokenStream;
+use proc_macro2::{Span, TokenStream};
```

Comment:
As you're now importing `Span`, can you please change the other occurrence in the code from `proc_macro2::Span` to just `Span`?